apps/www/src/lib/registry/default/example/DataTableReactiveDemo.vue

```diff
@@ -0,0 +1,273 @@
+<script setup lang="ts">
+import type {
+  ColumnDef,
+  ColumnFiltersState,
+  ExpandedState,
+  SortingState,
+  VisibilityState,
+} from '@tanstack/vue-table'
+import {
+  FlexRender,
+  getCoreRowModel,
+  getExpandedRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  useVueTable,
+} from '@tanstack/vue-table'
+import { ArrowUpDown, ChevronDown } from 'lucide-vue-next'
+
+import { h, ref } from 'vue'
+import DropdownAction from './DataTableDemoColumn.vue'
+import { Button } from '@/lib/registry/default/ui/button'
+import { Checkbox } from '@/lib/registry/default/ui/checkbox'
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+} from '@/lib/registry/default/ui/dropdown-menu'
+import { Input } from '@/lib/registry/default/ui/input'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/lib/registry/default/ui/table'
+import { valueUpdater } from '@/lib/utils'
+
+export interface Payment {
+  id: string
+  amount: number
+  status: 'pending' | 'processing' | 'success' | 'failed'
+  email: string
+}
+
+const data = ref<Payment[]>([
```

Comment:
The only confusion here is, in the new version they used `shallowRef` underhood but we pass data with `ref`

Isn't it better to use `shallowRef` instead of `ref` here to match their way?

Reply:
Yeah, makes sense, updated in [`ae4d5aa` (#744)](https://github.com/radix-vue/shadcn-vue/pull/744/commits/ae4d5aa1343e47a33b598a4da7f78605334cdee0). Honestly, there is no difference between `ref` and `shallowRef` when we pass them inside `useVueTable` because it is automatically transformed to `shallowRef` by @Tanstack/table package 🙂

Reply:
Right 👍  could you please mention it in the doc then

Reply:
Done in [`92801dc` (#744)](https://github.com/radix-vue/shadcn-vue/pull/744/commits/92801dce1c201f1d8cb92c31f824fee58b17a6fb)